### PR TITLE
Added the use of Process Explorer to capture child processes

### PR DESCRIPTION
**Pull request by Clare-Octopus**
Changed the page to add the use of Process Explorer to capture child processes as Task manager does not show child processes from Calamari and some of our engineers need that from our customers.

### CODE REVIEW

File: docs/support/process-dumps.md [hunk 1]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
-1. Find the relevant process. That will either be:
-   - **Octopus.Server.exe** or
-   - **Tentacle.exe** (and any **Calamari.exe** processes).
+1. Find the relevant process. In this case **Octopus.Server.exe**
 1. Right click on it and select **Create dump file**.
 1. Note where the file is saved (generally in your temp folder).
 
 :::warning
-When capturing a process dump for **Tentacle.exe**, please make sure to also capture any child **Calamari.exe** processes.
+When capturing a process dump for **Tentacle.exe**, please make sure to also capture any child **Calamari.exe** processes. To do this follow the process below:
```
Comment (harrisonmeister) [inline docs/support/process-dumps.md:31]:
```suggestion
When capturing a process dump for **Tentacle.exe**, please also capture any child **Calamari.exe** processes. To do this, follow the process below.
```

File: docs/support/process-dumps.md [hunk 2]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
+:::
+
+Task manager does not do a great job of capturing child processes such as the Calamari.exe process associated with Tentacle.exe. To allow the correct capture of a child process for a tentacle:
```
Comment (harrisonmeister) [inline docs/support/process-dumps.md:34]:
```suggestion
We recommend using [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer) to capture child processes associated with Tentacle.exe, such as the Calamari.exe process. To capture child processes for Tentacle:
```

File: docs/support/process-dumps.md (hunk 1 above)
Comment (Clare-Octopus) [reply docs/support/process-dumps.md:31]: Good change, I approve!

File: docs/support/process-dumps.md [hunk 3]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
+1. You will notice the program looks similar to task manager. Navigate to the **tentacle.exe** process in the list of tasks.
+1. Run the process that is causing the issue/fault in Octopus (ie run the deployment or task that is failing). 
+1. Once that task is running in Octopus go back to Process Explorer on the tentacle and you will now see the tentacle.exe process can be expanded to show the calamari process. This can be expanded further to see the powershell.exe processes associated with both the tentacle.exe and calamari.exe. 
+1. To capture the dump file for calamari.exe make sure your tentacle.exe process is expanded in process explorer and find calamari.exe.
+1. Right click on it and select `Create Dump` and then `Create Full Dump`
+1. Note where the file is saved (generally in your temp folder)
+
+:::hint
+Sometimes the deployment in Octopus can complete or error out too quickly which means you do not get chance to capture the failing process, if this is happening contact Support and they will suggest some workarounds to make that process last longer so you can capture the dump correctly.
```
[the remark this answers is not in the document]
Comment (Clare-Octopus) [reply docs/support/process-dumps.md:48]: Thats worded better!

File: docs/support/process-dumps.md [hunk 4]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
+1. Once that task is running in Octopus go back to Process Explorer on the tentacle and you will now see the tentacle.exe process can be expanded to show the calamari process. This can be expanded further to see the powershell.exe processes associated with both the tentacle.exe and calamari.exe. 
+1. To capture the dump file for calamari.exe make sure your tentacle.exe process is expanded in process explorer and find calamari.exe.
+1. Right click on it and select `Create Dump` and then `Create Full Dump`
+1. Note where the file is saved (generally in your temp folder)
+
+:::hint
+Sometimes the deployment in Octopus can complete or error out too quickly, which means you do not get a chance to capture the failing process. If this is happening [get in touch with us](https://octopus.com/support) and we can suggest some workarounds to make the process last longer so you can capture the dump correctly.
 :::
 
+![](images/processexplorer.png "width=500")
```
Comment (harrisonmeister) [inline docs/support/process-dumps.md:51]:
```suggestion
![Process explorer capturing child processes from Tentacle](images/processexplorer.png "width=500")
```

File: docs/support/process-dumps.md [hunk 5]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
+from Microsoft.
+1. Once installed you need to **run the program as an administrator** by right clicking on the relevent procexp.exe file and selecting `Run as Administrator`.
+1. Once opened you need to make sure the process tree is shown by clicking on the `View` menu on the top navigation bar and selecting `Show Process Tree`.
+1. You will notice the program looks similar to task manager. Navigate to the **tentacle.exe** process in the list of tasks.
+1. Run the process that is causing the issue/fault in Octopus (ie run the deployment or task that is failing). 
+1. Once that task is running in Octopus go back to Process Explorer on the tentacle and you will now see the tentacle.exe process can be expanded to show the calamari process. This can be expanded further to see the powershell.exe processes associated with both the tentacle.exe and calamari.exe. 
+1. To capture the dump file for calamari.exe make sure your tentacle.exe process is expanded in process explorer and find calamari.exe.
+1. Right click on it and select `Create Dump` and then `Create Full Dump`
```
Comment (harrisonmeister) [inline docs/support/process-dumps.md:44]:
```suggestion
1. Right click on it and select `Create Dump` and then `Create Full Dump`.
```

File: docs/support/process-dumps.md (hunk 2 above)
Comment (Clare-Octopus) [reply docs/support/process-dumps.md:34]: Much better, please commit this change.

File: docs/support/process-dumps.md [hunk 6]
```diff
@@ -19,18 +19,37 @@ Due to the nature of data contained in a process dump, we take great care in han
...
+1. Note where the file is saved (generally in your temp folder)
```
Comment (harrisonmeister) [inline docs/support/process-dumps.md:45]:
```suggestion
1. Note where the file is saved (generally in your temp folder).
```

**Pull request merged**